### DISCUSSION TITLE
MMath%40Waterloo/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#330] [Announcements]
# MMath%40Waterloo/

None

https://opencs.app/MMath%40Waterloo/

**Comment by ruoxining** [comment]
mmath是以前要求过s26w26吗 现在官网应该是s22w22了 (See https://uwaterloo.ca/future-graduate-students/programs/by-faculty/math/computer-science-master-math-mmath)
